app/workers/moderator/sink_articles_worker.rb

```diff
@@ -4,14 +4,15 @@ class SinkArticlesWorker
 
     sidekiq_options queue: :medium_priority, retry: 10
 
-    def perform(user_id)
```

Comment:
we actually can't do this because the reaction is destroyable (in which case the score should be lifted) and the sum of user reaction points + each articles score is correct. 

See https://github.com/forem/forem/blob/master/app/controllers/reactions_controller.rb#L133-L135 for a clear case where we can't assume this is "additive", we need to be able to remove the reaction as well.

Does passing user_id, reaction.score as a SinkArticles.for_user(user_id).by(reaction.score) permit this (rather than passing the reaction, or loading all reactions, we can pass the "subtract 25" or "add 25" value? Otherwise, support deletion by just fetching all user reactions and summing them, and fetching all article reactions and summing them, and assigning article score directly (just separating out the cross-summing over _all_ articles).

I think I'll have to revert 0f98a2011aad3de60b2b711bdea4ca99adfc00c3 and make sure this makes sense as initial written.